scripts/sct_get_centerline.py

```diff
@@ -125,6 +125,8 @@ def run_main():
     if method == 'viewer':
         # Manual labeling of cord centerline
         im_labels = _call_viewer_centerline(Image(fname_data), interslice_gap=interslice_gap)
+    if method == 'fitseg':
+        im_labels = Image(fname_data)
     else:
```

Comment:
change for elif method == 'optic' and add an else to catch error (better programming habit)